Common/src/config_structure.cpp

```diff
@@ -5159,11 +5158,7 @@ string CConfig::GetObjFunc_Extension(string val_filename) {
       case OUTFLOW_GENERALIZED:     AdjExt = "_chn";       break;
       }
     }
-    else {
-      if (DiscreteAdjoint) {
-        cout << endl << "Combined objective not yet compatible with discrete adjoint. Specify only one OBJECTIVE_FUNCTION." << endl << endl;
-        exit(EXIT_FAILURE);
-      }
+    else{
       AdjExt = "_combo";
     }
     Filename.append(AdjExt);
```

Comment:
This is not related with your changes :) Could you please change the indentation of GetnDim?